test_haystack/core/urls.py

```diff
@@ -1,28 +1,26 @@
 # encoding: utf-8
 
-from __future__ import absolute_import, division, print_function, unicode_literals
+from __future__ import unicode_literals
 
-from django.conf.urls import include, patterns, url
+from django.conf.urls import include, url
 from django.contrib import admin
 
+from haystack import views
 from haystack.forms import FacetedSearchForm
 from haystack.query import SearchQuerySet
-from haystack.views import FacetedSearchView, SearchView
 
 admin.autodiscover()
 
 
-urlpatterns = patterns('',
-    (r'^admin/', include(admin.site.urls)),
-)
+urlpatterns = [
+    url(r'^admin/', include(admin.site.urls)),
 
+    url(r'^$', views.SearchView(load_all=False), name='haystack_search'),
+    url(r'^faceted/$', views.FacetedSearchView(searchqueryset=SearchQuerySet().facet('author'),
```

Comment:
This should probably be indented to match the `FacetedSearchView(` so it's visually obvious which parameters are being passed to which function.